<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemManageWindow</class>
 <widget class="QWidget" name="SystemManageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f7fa;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">font-size: 24px; color: #2c3e50; padding: 20px; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>系统管理</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>60</number>
     </property>
     <property name="verticalSpacing">
      <number>40</number>
     </property>
     <!-- 修复边距设置：将rect替换为四个整数参数 -->
     <property name="leftMargin">
      <number>80</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>80</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnUserManage">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>160</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2980b9;
    border-radius: 6px;
    font-size: 18px;
    color: white;
    font-weight: bold;
    border: none;
}
QPushButton:hover {
    background-color: #3498db;
}</string>
       </property>
       <property name="text">
        <string>用户管理</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnOrgManage">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>160</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2980b9;
    border-radius: 6px;
    font-size: 18px;
    color: white;
    font-weight: bold;
    border: none;
}
QPushButton:hover {
    background-color: #3498db;
}</string>
       </property>
       <property name="text">
        <string>组织机构管理</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnPermissionManage">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>160</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2980b9;
    border-radius: 6px;
    font-size: 18px;
    color: white;
    font-weight: bold;
    border: none;
}
QPushButton:hover {
    background-color: #3498db;
}</string>
       </property>
       <property name="text">
        <string>权限管理</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnDictManage">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>160</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2980b9;
    border-radius: 6px;
    font-size: 18px;
    color: white;
    font-weight: bold;
    border: none;
}
QPushButton:hover {
    background-color: #3498db;
}</string>
       </property>
       <property name="text">
        <string>数据字典</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
